src/wmnds/components/content-card/_content-card.scss

```diff
@@ -1,3 +1,15 @@
+//accessibility styles
+@mixin accessibility-hover {
+  h2 {
+    text-decoration: underline;
```

Comment:
This should only happen when hovered according to: https://github.com/wmcadigital/wmn-design-system/issues/549#issuecomment-753901729

Reply:
None-issue...Resolved by speaking with @houbly.